src/sentry/static/sentry/app/components/events/groupingInfo.jsx

```diff
@@ -0,0 +1,335 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+import _ from 'lodash';
+
+import EventDataSection from 'app/components/events/eventDataSection';
+import SentryTypes from 'app/sentryTypes';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import LoadingError from 'app/components/loadingError';
+import {t} from 'app/locale';
+import KeyValueList from 'app/components/events/interfaces/keyValueList';
+
+import withApi from 'app/utils/withApi';
+
+const StyledGroupVariantList = styled('ul')`
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  font-size: 14px;
+  line-height: 18px;
+`;
+
+const StyledGroupVariantListItem = styled(({contributes, ...props}) => <li {...props} />)`
+  padding: 15px 0 20px 0;
+  margin-top: 15px;
+  border-top: 1px solid ${p => p.theme.borderLighter};
+  ${p => (p.contributes ? '' : 'color:' + p.theme.gray6)};
```

Comment:
Can be written as 

```jsx
color: ${p => (p.contributes ? null : p.theme.gray6)};
```

styled is smart here as well afaik

Reply:
I stole the pattern from the form system:

```javascript
const FieldControlState = styled(({flexibleControlStateSize, ...props}) =>
```

Reply:
Actually maybe I misunderstood the pattern there but this seems relatively clean to me, but I can change it.

Reply:
Either way, not super important. A lot of the little details I'm commenting on are stylistic and ideally we could lint for them